torchtitan/models/llama/__init__.py

```diff
@@ -48,4 +48,13 @@
         multiple_of=4096,
         rope_theta=500000,
     ),
+    "405B": ModelArgs(
+        dim=8192,
+        n_layers=80,
```

Comment:
Hmm, this config does not look right to me.

Reply:
it seems this is the same as 70B

Reply:
not sure if this is intentional and also not sure then if "wps: 539 mfu: 26.23%." is for 70B actually or not

Reply:
Now the config is the final one.